tests/queries/0_stateless/02232_non_negative_derivative.sql

```diff
@@ -0,0 +1,43 @@
+DROP TABLE IF EXISTS nnd;
+
+CREATE TABLE nnd
+(
+    id Int8, ts DateTime64(3, 'UTC'), metric Float64
+)
+ENGINE=MergeTree()
+ORDER BY id
+SETTINGS index_granularity = 8192;
```

Comment:
That's the default, no? An index_granularity may still be explicitly specified but it IMHO makes sense only if the result of the test depends on it.

Reply:
This is put here just to clearly specify test set-up